go/protocol/keybase1/extras.go

```diff
@@ -1266,6 +1266,7 @@ func (b TLFIdentifyBehavior) AlwaysRunIdentify() bool {
 	switch b {
 	case TLFIdentifyBehavior_CHAT_CLI,
 		TLFIdentifyBehavior_CHAT_GUI,
+		TLFIdentifyBehavior_FS_GUI,
```

Comment:
Seems like you didn't add `FS_GUI` to the function below this one, `CanUseUntrackedFastPath()`.  I'm not sure exactly what that does, but did you try it to see if it helps with the broken track reporting?

Reply:
I _think_ I tried. Will try again. I don't know what it is either